api.md

```diff
@@ -234,6 +236,33 @@ Response
 }
 ```
 
+<a name=device-authorized-post></div>
```

Comment:
I see that all of the other API sections do this, but I don't understand why there is a `</div>` instead of `</a>`

Reply:
Hah, nor do I. I just copied this from another section without noticing. Will fix.